Rakefile

```diff
@@ -151,6 +151,42 @@ namespace :docs do
           f.puts("_#{method.tag(:deprecated).text}_")
         end
       end
+
+      f.puts
+      f.puts("## ViewComponent::TestHelpers")
+
+      registry.
```

Comment:
Does it make sense to start thinking about extracting a documentation generator class since we repeat this logic a few times?

Reply:
Totally. We have similar code in Primer ViewComponents as well that I think we should consider abstracting...